src/tools/stmo.md

```diff
@@ -239,47 +168,22 @@ LIMIT
-specified limit. Clicking on the 'Save' button will then save the query, using
-the entered parameter value as the default.
+After you have replaced the hard-coded limit value with `{{country_count}}`, an input field appears above the bar chart. If you type a numeric value in the input field and click 'Execute', the query runs with the specified limit. Click 'Save' to save the query. The query applies the parameter value that you typed as the default.
 
-#### Creating A Dashboard
+#### Create A Dashboard
 
-Now we can create a dashboard to display our visualization. Do this by clicking
-on the 'Dashboards' drop-down near the top left of the page, and then clicking
-the 'New Dashboard' option. Choose a name for your dashboard, and you will be
-brought to a mostly empty page. Clicking on the '...' button near the top right
-of the page will give you the option to 'Add Widget'. This displays the
-following dialog:
+You can create a dashboard to display visualization by selecting 'New Dashboard' from the 'Dashboards' dropdown that is located at the top left of the page. Choose a name for your dashboard and an almost empty page appears. Click the '...' button near the top right of the page to selectn'Add Widget'. The 
+following dialog box appears:
 
 ![Add Widget](../assets/STMO_screenshots/add_widget.png "Add Widget")
 
-The "Search a query by name" field is where you can enter in the unique prefix
-used in your query name to find the query you created. This will not yet work,
-however, because your query isn't published. Publishing a query makes it show
-up in searches and on summary pages. Since this is only an exercise, we won't
-want to leave our query published, but it must be published briefly in order to
-add it to our dashboard. You can publish your query by returning to the query
-source page and clicking the "Publish" button near the top right of the screen.
-
-Once your query is published, it should show up in the search results when you
-type your unique prefix into the "Search a query by name" field on the "Add
-Widget" dialog. When you select your query, a new "Choose Visualization"
-drop-down will appear, allowing you to choose which of the query's
-visualizations to use. Choose the bar chart you created and then click on the
-"Add to Dashboard" button. Voila! Now your dashboard should have a bar chart,
-and you should be able to edit the `country_count` value and click the refresh
-button to change the number of countries that show up on the chart.
+Type a unique prefix that has previously been used in a query name in the "Search a query by name" field to locate the query that you have created. You cannot yet execute this query because the query has not yet been published. As soon as you publish a query, you can search on the summary pages. Even though this is only an exercise, the query must be published briefly and then added to the dashboard. You can publish your query by clicking "Publish" on the query source page.
```

Comment:
I think some of the meaning and explanation was lost here-- I think it might be best to revert this back to the original text, as I think it's relatively easy to understand.

Reply:
Not sure how best to do this - can we please discuss